packages/neo-one-node-vm/src/opcodes.ts

```diff
@@ -473,6 +473,31 @@ const functionCallIsolated = ({ name }: { readonly name: OpCode }): OpCreate =>
   },
 });
 
+const newArrayOrStruct = ({ name }: { readonly name: 'NEWARRAY' | 'NEWSTRUCT' }) =>
+  createOp({
+    name,
+    in: 1,
+    out: 1,
+    invoke: ({ context, args }) => {
+      let results;
+      if (args[0].isArray()) {
+        const array = args[0].asArray();
+        results = name === 'NEWARRAY' ? [new ArrayStackItem(array)] : [new StructStackItem(array)];
+      } else {
+        const count = vmUtils.toNumber(context, args[0].asBigInteger());
+
+        if (count > MAX_ARRAY_SIZE) {
+          throw new ContainerTooLargeError(context);
+        }
+
+        const fill = _.range(0, count).map(() => new BooleanStackItem(false));
+        results = name === 'NEWARRAY' ? [new ArrayStackItem(fill)] : [new StructStackItem(fill)];
+      }
+
+      return { context, results };
+    },
+  });
+
 const JMP = jump({ name: 'JMP' });
```

Comment:
I guess my below complaint is mute if its just the standard pattern for these?

Reply:
*shrugs*, yeah I was just matching the style of other opcodes.